src/skin/legacyskinparser.cpp

```diff
@@ -553,6 +554,8 @@ QList<QWidget*> LegacySkinParser::parseNode(const QDomElement& node) {
         result = wrapWidget(parseEffectName(node));
     } else if (nodeName == "EffectSelector") {
         result = wrapWidget(parseEffectSelector(node));
+    } else if (nodeName == "EffectParameterKnobComposed") {
+        result = wrapWidget(parseEffectParameterKnobComposed(node));
```

Comment:
The old Knob widget uses a series of images whereas the new KnobComposed widget uses a stationary background image and a single foreground image that is rotated.